blind_75/flatten_2d_vector.py

```diff
@@ -0,0 +1,72 @@
+# # https://leetcode.com/problems/flatten-2d-vector/
+
+# Design an iterator to flatten a 2D vector. It should support the next and 
+# hasNext operations.
+
+# Implement the Vector2D class:
+
+# Vector2D(int[][] vec) initializes the object with the 2D vector vec.
+# next() returns the next element from the 2D vector and moves the pointer one 
+# step forward. You may assume that all the calls to next are valid.
```

Comment:
Fascinating assumption. I would have still given an `IndexError` when reaching the end of the 2d matrix in an interview context.